### PR TITLE
de-nosing amo.  search to translations (& zadmin).

### PR DESCRIPTION
**Pull request by eviljeff**
The penultimate commit!

### CODE REVIEW

File: src/olympia/translations/tests/test_models.py [hunk 1]
```diff
@@ -603,62 +604,64 @@ class NoLinksTranslationTest(BaseTestCase):
...
         # Text link, markup link, allowed tags, forbidden tags and bad markup.
         s = (u'a <a href="http://example.com">link</a> with markup, a text '
              u'http://example.com link, <b>with allowed tags</b>, '
              u'<script>forbidden tags</script> and <http://bad.markup.com')
         x = NoLinksTranslation(localized_string=s)
-        eq_(x.__html__(), u'a  with markup, a text  link, '
-                          u'<b>with allowed tags</b>, '
-                          u'&lt;script&gt;forbidden tags&lt;/script&gt; and')
+        assert x.__html__() == (
+            u'a  with markup, a text  link, '
```
Comment (EnTeQuAk) [inline src/olympia/translations/tests/test_models.py]: double whitespace!

Comment (EnTeQuAk) [inline src/olympia/translations/tests/test_models.py]: Oh wait, that's unrelated and actually testing the code. Ignore that, that's another bug then :disappointed:

**Comment by EnTeQuAk** [comment]
r+

File: src/olympia/translations/tests/test_models.py (hunk 1 above)
Comment (eviljeff) [inline src/olympia/translations/tests/test_models.py]: I would have assumed (if I'd noticed) that it was deliberating checking we didn't squish extra whitespace.

Comment (EnTeQuAk) [inline src/olympia/translations/tests/test_models.py]: hmm, could be :laughing: 

I was so happy to have found that and then noticed that it's not really something to fix

**Pull request merged**